pkg/network/node/ovscontroller.go

```diff
@@ -199,16 +199,15 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 	// eg, "table=90, priority=100, ip, nw_dst=${remote_subnet_cidr}, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31], set_field:${remote_node_ip}->tun_dst,output:1"
 	otx.AddFlow("table=90, priority=0, actions=drop")
 
-	// Table 100: egress routing; edited by SetNamespaceEgress*()
-	otx.AddFlow("table=100, priority=300,udp,udp_dst=%d,actions=drop", vxlanPort)
-	otx.AddFlow("table=100, priority=200,tcp,tcp_dst=53,nw_dst=%s,actions=output:2", oc.localIP)
-	otx.AddFlow("table=100, priority=200,udp,udp_dst=53,nw_dst=%s,actions=output:2", oc.localIP)
```

Comment:
sorry, should have caught this before; these rules should stay in table 100, because we want them to bypass both egress IPs and EgressNetworkPolicy.

well... probably the DNS rules no-ops now anyway since we don't do DNS to the node IP any more in 4.x... but if someone is relying on this, we don't want to break it. (That is, if someone has configured pod DNS to talk to the node IP, and also has an EgressNetworkPolicy saying "Deny 0.0.0.0/0", then DNS to the node IP is still expected to work.)

Reply:
That will need a dedicated table 99 in that case. The reason I say that is because table 100 will now have priorities between 0 and number of rules specific in a egress network policy object. We won't be able to use a deterministic priority which overrides all rules, since we don't know what the max priority for all rules could be.